exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Exceptions.kt

```diff
@@ -37,4 +37,14 @@ class ExposedSQLException(cause: Throwable?, val contexts: List<StatementContext
 
 class UnsupportedByDialectException(baseMessage: String, val dialect: DatabaseDialect) : UnsupportedOperationException(baseMessage + ", dialect: ${dialect.name}.")
 
+/**
+ * DuplicateColumnException is thrown :
+ *
+ * When you attempt to create a table with multiple columns having the same name.
+ * When you replace a column of a table so that you define multiple columns having the same name.
+ *
+ * @param columnName the duplicated column name
+ */
+class DuplicateColumnException(columnName: String) : ExceptionInInitializerError("Duplicate column name : $columnName")
```

Comment:
I advise replacing `ExceptionInInitializerError` with `SQLException` as `Error` is rarely caught by applications and could fail the whole app. But I agree that `ExceptionInInitializerError` is better by semantic.